Il2CppInspector.Common/Reflection/TypeInfo.cs

```diff
@@ -32,6 +32,13 @@ private class TypeArgumentsComparer : EqualityComparer<TypeInfo[]>
         }
 
 
+        // This property exposes all types which have been generated directly from this one.
+        public IEnumerable<TypeInfo> CachedGeneratedTypes {
+            get {
+                return genericTypeInstances?.Values ?? Enumerable.Empty<TypeInfo>();
+            }
+        }
```

Comment:
Use lambda syntax

Reply:
This gets deleted in 067fe1f4a0b77752ed8f6d4ffe14138dbc4adb8b.